packages/vincent-tools/vincent-tool-erc20-approval/tools/scripts/deploy-lit-action.js

```diff
@@ -0,0 +1,73 @@
+const fs = require('fs');
+const path = require('path');
+const fetch = require('node-fetch');
+const FormData = require('form-data');
+
+const dotenvx = require('@dotenvx/dotenvx');
+
+// Load environment variables
+dotenvx.config({ path: path.join(__dirname, '../../../../../.env') });
+
+// Get Pinata JWT from environment variable
+const PINATA_JWT = process.env.PINATA_JWT;
+if (!PINATA_JWT) {
+  throw new Error('PINATA_JWT environment variable is not set in root .env file');
+}
+
+(async () => {
+  try {
+    const outputFile = 'vincent-tool-wrapped.js';
+
+    const generatedDir = path.join(__dirname, '../../src/generated');
+    const filePath = path.join(generatedDir, outputFile);
+    if (!fs.existsSync(filePath)) {
+      throw new Error(
+        `Bundled Lit Action code string not found at ${filePath}. Please run pnpx nx run vincent-tool-uniswap-swap:action:build first.`,
+      );
+    }
+    const litActionCodeString = require(filePath);
+
+    console.log(`Deploying ${outputFile} to IPFS...`);
+    const ipfsCid = await uploadToIPFS(outputFile, litActionCodeString.code);
+
+    const cidJsPath = path.join(generatedDir, 'vincent-tool-ipfs-cid.js');
+    const cidJsContent = `const vincentToolIpfsCid = '${ipfsCid}';\n\nmodule.exports = {\n  vincentToolIpfsCid,\n};\n`;
```

Comment:
See comment on other `deploy-lit-action` script.

Also, we need to get move these script(s) to be exported from the `vincent-tools-sdk` so that all of the tools/policies that bundle use a single source of truth -- otherwise we'll end up having to edit this script in many places any time we make a change to how we deploy the LIT actions